src/catalog/postgres/builder.cpp

```diff
@@ -765,12 +765,8 @@ IndexSchema Builder::GetProcNameIndexSchema(db_oid_t db) {
                        parser::ColumnValueExpression(db, PRO_TABLE_OID, PRONAME_COL_OID));
   columns.back().SetOid(indexkeycol_oid_t(2));
 
-  columns.emplace_back("proallargs", type::TypeId::VARBINARY, MAX_NAME_LENGTH, false,
-                       parser::ColumnValueExpression(db, PRO_TABLE_OID, PROALLARGTYPES_COL_OID));
-  columns.back().SetOid(indexkeycol_oid_t(3));
-
-  // Unique, not primary
-  IndexSchema schema(columns, storage::index::IndexType::BWTREE, true, false, false, true);
+  // Non-Unique, not primary
+  IndexSchema schema(columns, storage::index::IndexType::BWTREE, false, false, false, false);
```

Comment:
I'm honestly not super confident on these boolean parameters, if someone could double check them.

Reply:
Those look reasonable.